challenges/stats_preview_card_component/index.html

```diff
@@ -30,33 +30,74 @@
+          <img
+            class="block rounded-2xl rounded-b-none bg-cover bg-center sm:w-full sm:rounded-none sm:rounded-r-2xl sm:object-cover sm:object-center"
+            src="../images/image-header-mobile.jpg"
+            alt=""
+          />
+        </div>
+        <main
+          class="p-4 text-white sm:order-1 sm:flex-shrink sm:flex-grow sm:basis-2/4 sm:p-8 sm:text-left"
         >
+          <h1 class="text-2xl font-bold sm:mt-0 sm:text-2xl">
+            Get<span class="text-purple-600"> insights </span>that help your
+            business grow.
+          </h1>
+          <p class="mt-4 text-gray-400 sm:mt-4 sm:text-xs">
+            Discover the benefits of data analytics and make better decisions
+            regarding revenue, customer experience, and overall efficiency.
+          </p>
+          <section
+            class="mt-10 flex flex-col gap-6 pb-5 sm:mt-6 sm:flex-row sm:pb-0"
+          >
+            <div>
+              <h3 class="text-2xl font-bold">10k+</h3>
+              <p class="mt-2 text-sm uppercase text-gray-400">companies</p>
+            </div>
+            <div>
+              <h3 class="text-2xl font-bold">314</h3>
+              <p class="mt-2 text-sm uppercase text-gray-400">templates</p>
+            </div>
+            <div>
+              <h3 class="text-2xl font-bold">12M+</h3>
+              <p class="mt-2 text-sm uppercase text-gray-400">queries</p>
+            </div>
+          </section>
+        </main>
       </div>
-      <div>
-        for
-        <a
-          class="text-optimum-blue hover:text-optimum-darkblue"
-          href="https://github.com/optimumBA"
-          target="_blank"
-          >Optimum BH</a
-        >
-        Internship.
-      </div>
+      <div class="credits">
+        <div>
+          <a
+            class="text-optimum-blue hover:text-optimum-darkblue"
+            href="https://www.frontendmentor.io/challenges/stats-preview-card-component-8JqbgoU62"
+            target="_blank"
+            >Frontend Mentor challenge</a
+          >
+        </div>
+        <div class="text-white">
+          for
+          <a
+            class="text-optimum-blue hover:text-optimum-darkblue"
+            href="https://github.com/optimumBA"
+            target="_blank"
+            >Optimum BH</a
+          >
+          Internship.
+        </div>
 
-      <div class="mt-3">
-        <a
-          class="text-optimum-blue hover:text-optimum-darkblue"
-          href="https://github.com/optimumBA/frontend_mentor_challenges/tree/stats_preview_card_component/challenges/stats_preview_card_component"
-          >Source code</a
-        >
+        <div class="mt-3 text-white">
+          <a
+            class="text-optimum-blue hover:text-optimum-darkblue"
+            href="https://github.com/optimumBA/frontend_mentor_challenges/tree/stats_preview_card_component/challenges/stats_preview_card_component"
```

Comment:
Put your repo here so when anyone clicks on "Source code" link it takes them to your code.

Reply:
Done :+1: , thank you for the review @almirsarajcic